### PR TITLE
Rename section 1.1 "Cumulative Layout Shift".

### PR DESCRIPTION
**Pull request by skobes**
Some paragraphs that introduce the idea of a cumulative score are moved
into this section as well.

***
<a href="https://pr-preview.s3.amazonaws.com/skobes/layout-instability/pull/58.html" title="Last updated on Jul 31, 2020, 8:54 PM UTC (3ebd643)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/layout-instability/58/5a5126f...skobes:3ebd643.html" title="Last updated on Jul 31, 2020, 8:54 PM UTC (3ebd643)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -95,19 +95,17 @@ the severity of layout instability at a particular time.  The method of computin
...
-
-End of session signal {#end-of-session}
+Cumulative Layout Shift (CLS) {#cumulative-layout-shift}
 ---------------------------------------
 
 <em>This section is non-normative.</em>
 
-The developer can compute a cumulative layout shift score by summing the layout
-shift values as they are reported to the observer.
+The developer can use the layout shift values that are reported by this API
```
Comment (npm1) [inline index.bs:103]: Can we link to which layout shift values we're referring to here? i.e. link to 'value'.

File: index.bs [hunk 2]
```diff
@@ -95,19 +95,17 @@ the severity of layout instability at a particular time.  The method of computin
...
+to compute a cumulative score (the "cumulative layout shift score"),
```
Comment (npm1) [inline index.bs:104]: Perhaps we can phrase this as a definition: the <dfn>cumulative layout shift</dfn> of a page is...

This way it can be reference elsewhere via <a>.

File: index.bs (hunk 1 above)
Comment (skobes) [reply index.bs]: Done

File: index.bs (hunk 2 above)
Comment (skobes) [reply index.bs]: Done

**Comment by skobes** [comment]
Noticed another issue which is that we did not distinguish DCLS and CLS as in the explainer (https://github.com/WICG/layout-instability#cumulative-scores).  Latest changes address this.

File: index.bs [hunk 3]
```diff
@@ -95,23 +95,38 @@ the severity of layout instability at a particular time.  The method of computin
...
+API does not expose these values.)
+
+* The <dfn export>document cumulative layout shift (DCLS) score</dfn> is the
+    sum of every <a>layout shift value</a> that is reported inside a single
+    <a href="https://html.spec.whatwg.org/multipage/browsers.html#browsing-context">browsing context</a>.
+    (The DCLS score does not account for layout instability inside descendant browsing contexts.)
+
+* The <dfn export>cumulative layout shift (CLS) score</dfn> is the sum of every
+    <a>layout shift value</a> that is reported inside a <a>top-level browsing context</a>,
+    plus a fraction of each <a>layout shift value</a> that is reported inside any
```
Comment (npm1) [inline index.bs:118]: This sentence is a bit too long and hard to follow. We could consider splitting this into a tiny mini-algorithm. Also, would be nice to include a link for `descendant`, so as to make it clear from that link that we're talking about the full tree and not just immediate descendants.

File: index.bs [hunk 4]
```diff
@@ -95,23 +95,38 @@ the severity of layout instability at a particular time.  The method of computin
...
+    descendant browsing context, where that fraction, for each such occurrence, is
+    the fraction of the top-level viewport that is occupied by the viewport of the
```
Comment (npm1) [inline index.bs:120]: We've received feedback in the past (Paint Timing I think?) that viewport is too ambiguous and should not be used, opting for more precise terms. I'm not sure what's the correct language to use here though, so we can leave this for a followup. For now, at least we can include links as we do in other mentions of viewport in this spec.

File: index.bs [hunk 5]
```diff
@@ -95,23 +95,38 @@ the severity of layout instability at a particular time.  The method of computin
...
+The cumulative layout shift score is expected to have a general correspondence to
+the severity of layout instability for the lifetime of a page.
 
-A "final" score for the user's session can be reported by listening to the
-<a href="https://developers.google.com/web/updates/2018/07/page-lifecycle-api#event-visibilitychange">visibilitychange event</a>,
-and taking the value of the cumulative layout shift score at that time.
+The developer can use this API to compute the DCLS or CLS scores,
+by summing the values as they are reported to the observer,
+and taking the "final" score at the time of the
+<a href="https://developers.google.com/web/updates/2018/07/page-lifecycle-api#event-visibilitychange">visibilitychange event</a>.
```
Comment (npm1) [inline index.bs:129]: This should link to https://w3c.github.io/page-visibility/#dom-document-onvisibilitychange instead.

File: index.bs [hunk 6]
```diff
@@ -95,23 +95,38 @@ the severity of layout instability at a particular time.  The method of computin
...
+to have a value to represent the total instability of the page for the period of
+time the user spends on it.
+
+To that end we propose two values that a user agent or a developer is able to compute to
+obtain such a representation.  (These definitions are non-normative, because the
+API does not expose these values.)
+
+* The <dfn export>document cumulative layout shift (DCLS) score</dfn> is the
+    sum of every <a>layout shift value</a> that is reported inside a single
+    <a href="https://html.spec.whatwg.org/multipage/browsers.html#browsing-context">browsing context</a>.
```
Comment (npm1) [inline index.bs:113]: nit: There should be a way to link this without the explicit href in bikeshed.

File: index.bs (hunk 3 above)
Comment (skobes) [reply index.bs]: Split into two definitions and added links.

File: index.bs (hunk 4 above)
Comment (skobes) [reply index.bs]: Done

File: index.bs (hunk 5 above)
Comment (skobes) [reply index.bs]: Done

File: index.bs (hunk 6 above)
Comment (skobes) [reply index.bs:113]: I tried to fix this but got lost in bikeshed errors.
